app/components/advertisement-tool.js

```diff
@@ -31,100 +31,90 @@ export default Component.extend({
       full: true,
     },
     {
-      title: 'StudentenWegwijzer',
-      link: 'https://www.studentenwegwijzer.nl/enschede/studentenverenigingen/alpha/',
-      image: 'advertisements/studenten-wegwijzer.png',
-      full: true,
+      title: 'BeBo Parket',
+      link: 'https://www.beboparket.nl/',
+      image: 'advertisements/BeBo.png',
     },
     {
-      title: 'InShared',
-      link: 'https://www.inshared.nl/',
-      image: 'advertisements/inshared.png',
+      title: 'Boeketcadeau',
+      link: 'https://www.boeketcadeau.nl/',
+      image: 'advertisements/Boeketcadeau.png',
     },
     {
-      title: 'Led24',
-      link: 'https://www.led24.nl/led-verlichting/led-tl-buis/',
-      image: 'advertisements/led24.png',
+      title: 'Baktotaal',
+      link: 'https://www.baktotaal.nl/',
+      image: 'advertisements/Bouwhuis.png',
```

Comment:
Filename of image is "Bouwhuis" whilst the title is "Baktotaal". Its technically correct because they both represent the same website, but a but confusing. I recommend renaming the image file to "Baktotaal.png"